Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "AWSTools"
 uuid = "83bcdc74-1232-581c-948a-f29122bf8259"
 authors = ["Invenia Technical Computing"]
-version = "1.8.0"
+version = "1.8.1"
```

Comment:
```suggestion
version = "1.9.0"
```

Reply:
MJ beat me to it, since `AWSS3` now uses `AWS.AWSConfig()` instead of `AWSCore.AWSConfig()` we'll need to do a minor version bump here.

Reply:
Done.
@mattBrzezinski is the PR good to be merged, or there are other things that need to be thought through?